test/test_page.rb

```diff
@@ -24,6 +24,10 @@
     assert_equal 'Bilbo-Baggins.md', page.path
     assert_equal :markdown, page.format
     assert_equal @wiki.repo.commits.first.id, page.version.id
+
+    assert_not_nil page.last_version
+    assert_equal page.versions.first.id, page.last_version.id
+    assert page.last_version.stats.files.map{|file_stats| file_stats[0]}.include? page.path
```

Comment:
It's because `#stats` return [this Struct](https://github.com/gollum/rugged_adapter/blob/master/lib/rugged_adapter/git_layer_rugged.rb#L145). The structure of the struct (pun intended) is not very transparent, which I guess is causing the confusion. Another legacy from `grit`...